packages/babel-compat-data/package.json

```diff
@@ -29,8 +29,8 @@
     "compat-data"
   ],
   "devDependencies": {
-    "@mdn/browser-compat-data": "^3.1.1",
+    "@mdn/browser-compat-data": "^3.3.4",
     "core-js-compat": "^3.9.0",
-    "electron-to-chromium": "1.3.672"
+    "electron-to-chromium": "^1.3.738"
```

Comment:
The reason for not using `^` was https://github.com/babel/babel/pull/8299, but since we know have a lockfile I think we can?

If we still need to pint it, I would prefer to do it in the root `package.json#resolution` so all the pinned compat data packages are in the same place.

Reply:
Since we are removing lockfile on e2e test, we can move it to `resolution`.